app/package.json

```diff
@@ -93,7 +94,7 @@
     "react-native-cloud-storage": "^2.2.2",
     "react-native-device-info": "^14.0.4",
     "react-native-dotenv": "^3.4.11",
-    "react-native-gesture-handler": "^2.22.1",
+    "react-native-gesture-handler": "2.24.0",
```

Comment:
upgrade gesture handler to be compatible with our version of react native

![versions](https://github.com/user-attachments/assets/b655830c-d52d-4acc-8655-76b85f51d860)